src/test/regress/expected/with_basics.out

```diff
@@ -1011,6 +1011,14 @@ LEFT JOIN
         WHERE d1.user_id = d2.user_id )) AS bar USING (user_id);
 ERROR:  cannot pushdown the subquery
 DETAIL:  Complex subqueries and CTEs cannot be in the outer part of the outer join
+-- some test  with failures
+WITH a AS (SELECT * FROM users_table LIMIT 10)
+	SELECT 1/(random() * 0) FROM users_table JOIN a USING (user_id);
+ERROR:  division by zero
+CONTEXT:  while executing command on localhost:xxxxx
+WITH a AS (SELECT * FROM users_table LIMIT 10)
+	SELECT 1/0 FROM users_table JOIN a USING (user_id);
+ERROR:  division by zero
```

Comment:
this error seems to come from the coordinator, maybe do `user_id/0`

Reply:
we have (random() * 0) test right below, but I can improve this as well

Reply:
right now the error comes from the planner I think, so this doesn't exercise any executor code

Reply:
> WITH a AS (SELECT * FROM users_table LIMIT 10)
	SELECT 1/(random() * 0) FROM users_table JOIN a USING (user_id);
..
NOTICE:  issuing SELECT ((1)::double precision OPERATOR(pg_catalog./) (random() OPERATOR(pg_catalog.*) (0)::double precision)) FROM (SELECT FROM (public.users_table_102013 users_table(user_id, "time", value_1, value_2, value_3, value_4) JOIN (SELECT intermediate_result.user_id, intermediate_result."time", intermediate_result.value_1, intermediate_result.value_2, intermediate_result.value_3, intermediate_result.value_4 FROM read_intermediate_result('5_1'::text, 'binary'::citus_copy_format) intermediate_result(user_id integer, "time" timestamp without time zone, value_1 integer, value_2 integer, value_3 double precision, value_4 bigint)) a USING (user_id))) worker_subquery
...
ERROR:  division by zero
CONTEXT:  while executing command on localhost:9701

This one comes from the executor? Anyway, I think `user_id/0` easier to understand so I'll do that